File: 04 - JavaScript/08-03-00-Paso-Por-Valor-ULP.js
// Funciones en JavaScript
// En JavaScript los valores simples se pasan por valor 
function cambiarValor(parametro){
    parametro = 20
}

// Llamamos a la funcion
let argumento = 10
console.log(`Antes de llamar funcion: ${argumento}`)
cambiarValor(argumento)
console.log(`Despues de llamar funcion: ${argumento}`)

File: 06 - Java/08-03-00-Paso-Por-Valor-ULP.java
public class ProgramaPrincipal {

    // Definimos la funcion
    static void cambiarValor(int parametro) {
        parametro= 20;
    }

    public static void main(String[] args){
        // Creamos las variables
        var argumento = 10;
        System.out.println("Antes llamar funcion: " + argumento);
        // Llamamos a la funcion
        cambiarValor(argumento);
        System.out.println("Despues llamar funcion: " + argumento);
    }
}